base/sysimg.jl

```diff
@@ -400,6 +400,24 @@ end
 INCLUDE_STATE = 3 # include = include_from_node1
 include("precompile.jl")
 
+@noinline function test_clone_f(a)
+    s = zero(eltype(a))
+    @inbounds @simd for i in 1:length(a)
+        s += a[i]
+    end
+    return s
+end
+
+@noinline function test_clone_g(a, n)
+    s = zero(eltype(a))
+    for i in 1:n
+        s += test_clone_f(a)
+    end
+    return s
+end
+
+test_clone_g(Float64[], 1)
```

Comment:
very true. Another nice one to add support for with this infrastructure is `@llvm.fma.f64` (e.g. move the https://github.com/JuliaLang/julia/blob/a204bfdb4c2968ad5184c6e75451b0060851bf3f/base/floatfuncs.jl#L222 hack into the compiler)